front/components/vaults/ContentActions.tsx

```diff
@@ -13,8 +13,11 @@ import { DocumentOrTableUploadOrEditModal } from "@app/components/data_source/Do
 import { MultipleDocumentsUpload } from "@app/components/data_source/MultipleDocumentsUpload";
 import { isFolder, isWebsite } from "@app/lib/data_sources";
 
-type ContentActionKey =
-  | "DocumentOrTableUploadOrEditModal"
+export type ContentActionKey =
+  | "EditDocument"
+  | "EditTable"
+  | "CreateDocument"
+  | "CreateTable"
```

Comment:
Do we really need 4 actions? Couldn't we deal with only 2? (DocumentUploadOrEdit and TableUploadOrEdit)

Reply:
not really needed indeed